charts/home-assistant/README.md

```diff
@@ -94,7 +95,8 @@ The following tables lists the configurable parameters of the Home Assistant cha
 | `hostMounts.mountPath`                          | The path at which to mount (optional; assumed same as hostPath)                                                                                                                                                                           | `nil`                               |
 | `hostMounts.type`                               | The type to mount (optional, i.e., `Directory`)                                                                                                                                                                                           | `nil`                               |
 | `extraEnv`                                      | Extra ENV vars to pass to the home-assistant container                                                                                                                                                                                    | `{}`                                |
-| `extraEnvSecrets`                               | Extra env vars to pass to the home-assistant container from k8s secrets - see `values.yaml` for an example                                                                                                                                | `{}`                                |
+| `extraEnvSecrets`                               | Extra env vars to pass to the home-assistant container from k8s secrets - see `values.yaml` for an example                                                                                                                                | `{}`                               |
+| `configSubPath`                                 | An optional subPath for the config volumeMount                           | ``                                |
```

Comment:
Should we also add this to the `values.yaml` file so folks know that it's an option that can be set?  Generally if something is a configurable value, it will have a stub (or commented stub) in `values.yaml`

Reply:
ok I can do that.